shared/menubar/icons.tsx

```diff
@@ -0,0 +1,22 @@
+import * as NotificationTypes from '../constants/types/notifications'
+import {isDarwin, isWindows, isLinux} from '../constants/platform'
+
+const getIcons = (iconType: NotificationTypes.BadgeType, isBadged: boolean, isSystemDarkMode: boolean) => {
```

Comment:
Why pass in `isSystemDarkMode` instead of importing it here?

Reply:
because this is used in the node and chrome contexts and i don't want to pull in all those deps

Reply:
and pulling it out of styles depends on timing which i don't want to race on

Reply:
Got it, thanks.